web-ui/src/app/components/header-3/header-3.component.html

```diff
@@ -0,0 +1 @@
+<h2><ng-content></ng-content></h2>
```

Comment:
Uhm? I think having a header3component (which is also empty and actually an `<h2>`) is a bit unnecessary.